KarateTraining/src/test/java/karate-config.js

```diff
@@ -0,0 +1,26 @@
+function fn() {
+  var env = karate.env; // get system property 'karate.env'
+  var jsonConfig = read('classpath:src/main/resources/qa.json');
```

Comment:
why do you put this file on main package? I would put this file at the root or inside the test package